From 8eac272ab4d36969b5a4a368e2b4d9e01382e10b Mon Sep 17 00:00:00 2001
From: Subbaraya Sundeep <sbhatta@marvell.com>
Date: Wed, 27 Mar 2019 11:51:09 +0530
Subject: [PATCH 0139/1921] octeontx2-af: Remove unnecessary error message

When a default DMAC rule cannot be installed by
rvu_npc_install_ucast_entry then it is not an error because
user might have called "ifconfig ethx hw ether" before
interface is up which is a valid scenario. Hence removed
the error message.

Change-Id: Ib5da3eba5c179e0dcf8255681327543d6b108134
Signed-off-by: Subbaraya Sundeep <sbhatta@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
index 7536d646298d..c2b0d7a74c97 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
@@ -433,8 +433,7 @@ void rvu_npc_install_ucast_entry(struct rvu *rvu, u16 pcifunc,
 	req.match_id = action.match_id;
 	req.flow_key_alg = action.flow_key_alg;
 
-	if (rvu_mbox_handler_npc_install_flow(rvu, &req, &rsp))
-		dev_err(rvu->dev, "could not install default rule\n");
+	rvu_mbox_handler_npc_install_flow(rvu, &req, &rsp);
 }
 
 void rvu_npc_install_promisc_entry(struct rvu *rvu, u16 pcifunc,
-- 
2.31.1

